packages/forms-web-app/src/public/resources/plans-drawings/upload-plans-drawings.html

```diff
@@ -0,0 +1,3 @@
+<p class="govuk-body">You must upload all of the plans, drawings and supporting documents that you submitted to the local planning authority with your planning application.</p>
```

Comment:
>with your planning application.</p>

Just wondering if the word planning should be removed here as its been removed pretty much everywhere else?